sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseListCriteriaBuilder.java

```diff
@@ -102,6 +106,22 @@
 					cb.isFalse(eventCountRoot.get(EventParticipant.DELETED))));
 			eventCountSq.select(cb.countDistinct(event.get(Event.ID)));
 			selectionList.add(eventCountSq);
+
+			// Latest sampleDateTime subquery
+			Subquery<Timestamp> latestSampleDateTimeSq = cq.subquery(Timestamp.class);
+			Root<Sample> sample = latestSampleDateTimeSq.from(Sample.class);
+			Path<Timestamp> sampleDateTime = sample.get(Sample.SAMPLE_DATE_TIME);
+			latestSampleDateTimeSq
+				.where(cb.equal(sample.get(Sample.UUID), joins.getSamples().get(Sample.UUID)), cb.isFalse(sample.get(Sample.DELETED)));
```

Comment:
Any reason you're using the UUID here and the ID in the count subquery? ID would generally be preferable because it's shorter